Docs/user_guide/getting_started.md

```diff
@@ -96,6 +96,15 @@ To run Slicer-4.11-2020-09-30 on older debian (e.g. debian 9) you may also need:
 #### ArchLinux
 ArchLinux runs the `strip` utility by default; this needs to be disabled in order to run Slicer binaries.  For more information see [this thread on the Slicer Forum](https://discourse.slicer.org/t/could-not-load-dicom-data/14211/5).
 
+#### Fedora
+Install the dependencies:
+
+    sudo dnf install mesa-libGLU libnsl
+
+The included libcrypto.so.1.1 in Slicer-4.11 is incompatible with the system libraries used by Fedora. The fix, until it is updated, is to move/remove the included libcrypto files:
```

Comment:
Could you list the Fedora version number here just for completeness?

Reply:
Sure, I'll add the version (35) as soon as I know which branch what info should go in. I'm pretty sure the dependencies and the libcrypto issue is the same for Fedora since version ~30, and most likely for Alma/Rocky Linux as well, but will leave it as Fedora 35

Reply:
Hi @pieper, do you have other requests for changes?